test/helpers/StoreAndHeadersFactory.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Wovnio\Test\Helpers;
+
+use Wovnio\Wovnphp\Store;
+use Wovnio\Wovnphp\Headers;
+
+class StoreAndHeadersFactory {
+  public static function fromFixture($fixture = 'default', $settingsOverwrite = array(), $envOverwrite = array()) {
```

Comment:
I think a Factory should be like

```
class StoreAndHeadersFactory {
  public function setSomething($option) { // could have many setSomething for each option like store, fixture, etc. Default values are set in constructor
    $this->$someOption = $option;
    return $this;
  }

  public function build() {
     // build store and header based on what are set
     ...

     return array($store, $headers)
  }
}
```

and being called like `new StoreAndHeadersFactory().setSomething($option).build()`

How do you think?

Reply:
This is not following best practice Factory Method. Or maybe you have a reference you can share? This approach also make it unreliable when used multiple time. I prefer the way it is now.

Reply:
indeed, it's Builder pattern. I think this make it clear that when you want to use different options, you have to set it up explicitly, and have just 1 `build()`, instead of having multiple methods for each type of build (from fixture, custom store, etc.)

Reply:
I prefer using Factory Method here.